test/ip-examples.js

```diff
@@ -1,6 +1,6 @@
 const log = console.log.bind (console)
-const { ipv4, ipv6 } = require ('../lib/host')
-const assert = require ('assert').strict
+import { ipv4, ipv6 } from '../lib/host'
```

Comment:
Btw. this file seems to be dead code currently, since `../lib/host.js` doesn't exist. I saw it exists in spec-url, so maybe this was before moving it? Should I delete it here?

Reply:
Ah cool, you're right. Yes, it can be removed!